### PR TITLE
Improvement - hover to highlight features

### PR DESCRIPTION
**Pull request by ddjnw1yu**
This is for the ticket - [Sidebar improvement - hover to show highlight features on map](https://www.wrike.com/open.htm?id=1408309810)

Removed the zoom-in marker action when hovering over a dataset.

Instead, all dataset-related features will now be highlighted.

Screenshot 2024-07-04 at 9 59 24 AM
Screenshot 2024-07-04 at 9 59 44 AM

### CODE REVIEW

**Comment by ddjnw1yu** [comment]
Added highlight functionality for scaffold and flatmap as required.

Screenshot 2024-07-05 at 3 02 36 PM

Screenshot 2024-07-05 at 3 02 11 PM

File: src/components/SplitFlow.vue [hunk 1]
```diff
@@ -258,16 +257,10 @@ export default {
...
-      if (!hoverEntries.length) {
-        this.hoveredMarkerDelay = setTimeout(() => {
-          EventBus.emit("markerUpdate");
-        }, 500)
-      } else {
-        clearTimeout(this.hoveredMarkerDelay)
-        EventBus.emit("markerUpdate");
-      }
+      const hoverAnatomies = data && data.anatomy ? data.anatomy : []
+      const hoverOrgans = data && data.organs ? data.organs : []
```
Comment (alan-wu) [inline src/components/SplitFlow.vue:261]: Missing semi-colons here.

File: src/mixins/ContentMixin.js [hunk 2]
```diff
@@ -486,6 +486,26 @@ export default {
...
     },
+    mapHoverHighlight: function (mapImp) {
+      const hoverAnatomies = this.settingsStore.hoverAnatomies
+      const hoverOrgans = this.settingsStore.hoverOrgans
+      if (hoverAnatomies.length || hoverOrgans.length) {
+        clearTimeout(this.hoverDelay)
+        if (this.multiflatmapRef || this.flatmapRef) {
+          mapImp?.zoomToFeatures(hoverAnatomies, { noZoomIn: true })
+        } else if (this.scaffoldRef) {
+          mapImp?.changeHighlightedByName(hoverOrgans, "", false)
```
Comment (alan-wu) [inline src/mixins/ContentMixin.js:497]: Missing semi-colons here.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Looks good to me. Can you add the semi-colons to some of the end of line please?

**Comment by ddjnw1yu** [comment]
Is there a time limit for the testing? Testing uses fixtures, no reason it will be aborted...

Screenshot 2024-07-10 at 11 21 17 AM

**Review by alan-wu — APPROVED** [review]
Looks good to me.

**Pull request merged**